diff -ru slapt-src-6020cb0/src/main.c slapt-src-6020cb0.new/src/main.c
--- slapt-src-6020cb0/src/main.c	2020-03-08 12:30:04.000000000 +0100
+++ slapt-src-6020cb0.new/src/main.c	2021-07-30 17:59:22.174667593 +0200
@@ -37,6 +37,7 @@
 
 static int show_summary(slapt_vector_t *, slapt_vector_t *, int, bool);
 static void clean(slapt_src_config *config);
+char command[12];
 
 void version(void)
 {
@@ -352,8 +353,10 @@
     /* now, actually do what was requested */
     switch (action) {
     case UPDATE_OPT:
+        strcpy( command, "SBoreplace" );
         if (!slapt_src_update_slackbuild_cache(config))
             exit(EXIT_FAILURE);
+        system(command);
         break;
 
     case FETCH_OPT:
